modules/ROOT/pages/6.3-release-notes.adoc

```diff
@@ -0,0 +1,704 @@
+
+The root cause was the insertion point being placed into a fake block caret, which was not considered by the logic for toggling the Quick Insert context toolbar.
+
+A fix was applied to the logic to ensure it could handle such cases, and prevent it from showing in this circumstance.
+
+
+=== When running on Firefox, the editor.selection.getRng() API was not returning expected values on hidden editors
+
+In contrast to other supported browsers, Firefox did not return the selected object or the selection range from iframes that are hidden by the `display: none` attribute.
+
+As a consequence, when a {productname} instance was running on Firefox, the `window.getSelection()` API for an iframe returned `null` if the iframe was hidden. This, further, resulted in the `editor.selection.getRng()` and `editor.selection.getBookmark()` APIs returning incorrect data from hidden editors when Firefox was the host browser.
+
+This has been fixed in {productname} 6.3. All browsers, including Firefox, now return the correct values even when the iframe is hidden.
+
+
+=== Dragging a noneditable element before or after another noneditable element now works correctly.
+
+Elements that had the `contenteditable="false"` attribute set could not be dragged to the left of an ancestor element.
+
+For example in the following
+
+[source,html]
+----
+<p>Text in a paragraph tag with a span of <span contenteditable="false">non-editable text</span> inside.</p>
+----
+
+the string _non-editable text_ could not be dragged to before the opening `+<p>+` tag.
+
+{productname} 6.3 can place a fake insertion point next to elements with a `contenteditable="false"` attribute when another element with a `contenteditable="false"` attribute is dragged, allowing the object to be moved before and after other such elements.
+
+
+=== The restored selection or location after a redo or undo action was not scrolled into view.
+
+Prior to the {productname} 6.3 release, executing an undo or redo function in the editor did not scroll the editor to the new insertion point or selection, should the undo or redo result in the insertion point or selection being out of the current viewport.
+
+Instead, users had to manually scroll to the insertion point or selection after issuing an undo or redo command that resulted in the insertion point or selection being out of view.
+
+As of the {productname} 6.3 release, the scroll function now repositions the insertion point to the last known undo or redo location.
+
+
+=== A newline could not be inserted after an inline element with a contenteditable="false" attribute was created.
+
+In previous versions of {productname}, a newline could not be inserted when the selection was restored from a ‘bookmark’ positioning after adding an inline element with a `contenteditable="false"` attribute.
+
+This was most commonly seen by pressing the *Return* (or *Enter*) key not moving the insertion point to a new line after a merge tag was created in the editor.
```

Comment:
We probably need a search and replace with a consistent capitalisation of Merge Tag

Reply:
> We probably need a search and replace with a consistent capitalisation of Merge Tag

Agreed.

That said, this particular instance is correct of itself. This _merge tag_ is referring to a `{{<string-here>}}` instance in a document. So, the denoting term is a common-or-garden-variety noun-phrase. So, no capitals unless the first word in the phrase is also the first word in a sentence.